languages.toml

```
@@ -489,6 +489,34 @@ args = { processId = "{0}" }
 name = "c-sharp"
 source = { git = "https://github.com/tree-sitter/tree-sitter-c-sharp", rev = "5b60f99545fea00a33bbfae5be956f684c4c69e2" }
 
+[[language]]
+name = "cel"
+scope = "source.cel"
+injection-regex = "cel"
+file-types = ["cel"]
+comment-token = "//"
+roots = []
+indent = { tab-width = 2, unit = "  " }
+
+[[grammar]]
+name = "cel"
+source.git = "https://github.com/bufbuild/tree-sitter-cel"
+source.rev = "9f2b65da14c216df53933748e489db0f11121464"
```

```suggestion
source = { git = "https://github.com/bufbuild/tree-sitter-cel", rev = "9f2b65da14c216df53933748e489db0f11121464" }
```

and same below, just to match the style we use for the rest of languages.toml